feat/new product overview pages (#646)
hirosystems/docs: docs/subnets/index.mdx

> **_NOTE:_**
>
> You can launch a subnet on Stacks mainnet today, but more work is needed to fully support subnets via Hiro tools. Reach out to us on Discord if you are interested in launching a subnet for your app.

<div class="flex justify-start my-8">
  <a href="/subnets/getting-started" class="pr-3">
    Get started with subnets <span classNam="ml-1 no-underline">→</span>
  </a>
</div>

### Guides

<div class="subSections my-8">
  <a href="/subnets/getting-started">
    <div class="subSectionTitle"><h3>Set up a subnet demonstration</h3><span>→</span></div>
    <p>Experiment with subnets with a basic NFT example.</p>
  </a>
  <a href="/subnets/event-dispatcher">
    <div class="subSectionTitle"><h3>Configure developer tools</h3><span>→</span></div>
    <p>Set up developer tools and APIs for a specific subnet.</p>
  </a>
</div>

<div class="subSections my-8 w-[50%]">
  <a href="/subnets/guides/subnet-participants">
    <div class="subSectionTitle"><h3>Dive into subnet architecture</h3><span>→</span></div>
    <p>Understand subnet configuration with this primer on architecture.</p>
  </a>
</div>

---

> Need help building with Subnets? Reach out to us on the `#builder-general` channel on Discord under Hiro Developer Tools section. We also host a weekly office hours on Discord every Monday at 11am ET.

Reviewer: ```suggestion
Need help building with Subnets? Reach out to us on the `#builder-general` channel on Discord under the [Hiro Developer Tools section](https://stacks.chat/). There’s also a [weekly office hours](https://events.stacks.co/event/HD16484710) on Discord every Thursday at 11am ET.
```
